### ISSUE TITLE
Obsidian freezes with floating window and obsidian advanced slides preview upon starting

### ISSUE THREAD

**Issue by lenalebt** [MSzturc/obsidian-advanced-slides#258]
**Describe the bug**
Obsidian freezes when starting the app if a slide preview is open, and a floating window as well.

It's pretty reproducible for me, I need to manually edit `workspace.json` and delete the config around the floating window to get it running again.

**Samples to Reproduce**

- Open slide preview for advanced slides
- Open a floating window, so, a file in an additional obsidian window of the same instance
- close obsidian
- reopen obsidian
- obsidian freezes

**Expected behavior**

- Obsidian does not freeze when starting

I would love to further debug this, but I have no clue how to get more info out of the frozen Obsidian :sweat_smile:

**Comment by Cratesmith** [comment]
I'm experiencing this too. It's far more reproducable if you have the slides preview in split mode.

That said it does happen in tab mode as well. Once it starts happening the only way to be able to use a slide preview again seems to be to clear workspace.json

**Comment by patrick-hilo** [comment]
Yes, I encountered that problem as well. Closing that window and then restarting obsidian resolves that problem.
